packages/ibm-products/src/components/EditTearsheet/EditTearsheet.tsx

```diff
@@ -27,11 +37,95 @@ export const FormNumberContext = createContext(0);
 
 // Default values for props
 const defaults = {
-  verticalPosition: 'normal',
-  influencerWidth: 'narrow',
-  sideNavAriaLabel: 'Side navigation',
+  verticalPosition: 'normal' as const,
+  influencerWidth: 'narrow' as const,
+  sideNavAriaLabel: 'Side navigation' as const,
 };
 
+// Note that the descriptions here should be kept in sync with those for the
+// corresponding props for TearsheetNarrow and TearsheetShell components.
+interface EditTearsheetProps extends PropsWithChildren {
```

Comment:
Hey ,
I think all props except `onRequestSubmit` are optional. Could you please check that?

Reply:
Fixed!